### PR TITLE
Update code to follow code convention

### PR DESCRIPTION
**Pull request by puerile22**
_**Related items**_

- [[Ticket](link)](https://motivemetrics.atlassian.net/browse/RD-242?atlOrigin=eyJpIjoiMjY2MDkzN2NmNDUyNGYwOWFhNTZmY2MwZGFlMTAzZjciLCJwIjoiaiJ9)

_**Problem**_

Problem.
Google ads data repo is not following new code convention.

_**Solution**_

Update code to follow new code convention.

_**Other changes**_

-None

_**Testing**_

- No testing are done.

_**Code quality**_

- how did you verified code quality?
  - ran linters (sublimelinter flake8)

_**What can be broken**_

- Google-ads-data package could be broken.

_**Readme Updates**_

- Are there any updates that should be made to the readme?
  For example: Is there anything that you did manually to get this code working?
  - no

### CODE REVIEW

**Review by mkarbouski — DISMISSED** [review]
Again nice job!

**Review by rsen-motivemetrics — CHANGES_REQUESTED** [review]

File: google_ads_data/account_utils.py [hunk 1]
```diff
@@ -39,92 +60,145 @@ def cust_id_to_account(custId: str, env: str = "prod") -> dict:
...
-        The Google Ads ``customer.id`` resource for the account.
+        Returns:
+            cust_id (str): The Google Ads ``customer.id`` resource for the account.
 
-    Example::
-        custId = account_name_to_cust_id("car.com", custname="autoweb")
     """
-    account = account_name_to_account(acctname, custname, env)
+    account = account_name_to_account(account_name, cust_name, env)
     if account is None or "data" not in account or "customerId" not in account["data"]:
```
[the remark this answers is not in the document]
Comment (rsen-motivemetrics) [reply google_ads_data/account_utils.py:183]: But I see a lot of improvements @puerile22 well done 🥇

**Review by naseka-mm — DISMISSED** [review]

**Review by ikhodas — APPROVED** [review]
Just work with Roman review advices, after this you can merge, nice job!

File: google_ads_data/account_utils.py [hunk 2]
```diff
@@ -30,7 +40,18 @@ def appx_mongo_db(env: str = "prod") -> pymongo.database.Database:
     return db
 
 
-def cust_id_to_account(custId: str, env: str = "prod") -> dict:
+def cust_id_to_account(cust_id: str, env: str = "prod") -> dict:
```
[the remark this answers is not in the document]
Comment (puerile22) [reply google_ads_data/account_utils.py:43]: I think `env` is pretty descriptive enough, right? I have not used Enum before but I'll try update this.

**Review by mkarbouski — APPROVED** [review]
I love the progress that we have!

**Review by ikhodas — APPROVED** [review]
Great Job

**Review by GermanParra — APPROVED** [review]

**Review by rsen-motivemetrics — APPROVED** [review]

**Pull request merged**